lib/matplotlib/colors.py

```diff
@@ -1152,7 +1152,7 @@ def __init__(self, vmin=None, vmax=None, clip=False):
         self.vmin = _sanitize_extrema(vmin)
         self.vmax = _sanitize_extrema(vmax)
         self.clip = clip
-        self._scale = scale.LinearScale(axis=None)
+        self._scale = None  # will default to LinearScale for colorbar
```

Comment:
Note downstream libraries will likely derive from Normalize, so scale should be None so their norm gets used...

Reply:
@dstansby can you see if this version works?

Reply:
Yep, seems to work well with this version.

Reply:
Any chance of a formal review?